pages/osx/hashcat.md

```diff
@@ -0,0 +1,24 @@
+# hashcat
+
+> Password recovery utility
+> More information: <https://hashcat.net/wiki/>.
+
+- Wordlist
+
+`hashcat -a 0 -m 400 example400.hash example.dict`
+
+- Wordlist + Rules
+
+`hashcat -a 0 -m 0 example0.hash example.dict -r rules/best64.rule`
+
+- Brute-Force
+
+`hashcat -a 3 -m 0 example0.hash ?a?a?a?a?a?a`
```

Comment:
```suggestion
- Brute-Force:

`hashcat --attack-mode={{3}} --hash-type={{0}} {{example0.hash}} "?a?a?a?a?a?a"`
```

Almost the same issue as above.
Optional fix: broken literal: quoted glob-like literal expected.